src/content/translations/id/developers/docs/nodes-and-clients/index.md

```diff
@@ -166,7 +166,7 @@ Mode sinkronisasi mewakili pendekatan berbeda terhadap proses ini dengan berbaga
 
 Gambaran umum pendekatan sinkronisasi yang digunakan dalam klien yang siap untuk Jaringan Utama:
 
-##### Sinkonisasi penuh {#full-sync}
+##### Sinkonisasi penuh
```

Comment:
Should we remove custom header id's from all H5s on the english versions (in a separate PR) to avoid this in the future?

Reply:
On my list but yes feel free.